generic/tests/ioquit.py

```diff
@@ -26,16 +31,32 @@ def run(test, params, env):
     session2 = vm.wait_for_login(timeout=login_timeout)
 
     bg_cmd = params.get("background_cmd")
-    error.context("Add IO workload for guest OS.", logging.info)
+    error_context.context("Add IO workload for guest OS.", logging.info)
     session.cmd_output(bg_cmd, timeout=60)
 
-    error.context("Verify the background process is running")
+    error_context.context("Verify the background process is running")
     check_cmd = params.get("check_cmd")
     session2.cmd(check_cmd, timeout=360)
 
-    error.context("Sleep for a random time", logging.info)
+    error_context.context("Sleep for a random time", logging.info)
     time.sleep(random.randrange(30, 100))
     session2.cmd(check_cmd, timeout=360)
 
-    error.context("Kill the VM", logging.info)
+    error_context.context("Kill the VM", logging.info)
     vm.process.close()
+
+    error_context.context("Check img after kill VM", logging.info)
+    base_dir = data_dir.get_data_dir()
+    image_name = params.get("image_name")
+    image = qemu_storage.QemuImg(params, base_dir, image_name)
+    try:
+        image.check_image(params, base_dir)
+    except exceptions.TestWarn as e:
```

Comment:
It should raise other type of exception (e.g. `ClusterLeakError`) rather than `TestWarn`, so here, I suggest capturing all the exceptions (`Exception`) as a workaround.